blocks/media/media.js

```diff
@@ -174,42 +183,6 @@ function createMediaDots(totalCards, maxAllowedCards, dots) {
   }
 }
 
-function startCarousel(block) {
```

Comment:
not needed anymore

Reply:
@ravverma auto scroll is disabled now onwards ?
Is it at par with original website ?

Reply:
Auto scroll behavior moved here https://github.com/aemsites/icicidirect/pull/266/files#diff-06f3a957cd23bb3d1cacf9972d09a491cf63c38c04697bc258e3e3f4980164bfR215
and it is same as current site..stopped at end